backend/controller/sql/queries.sql

```diff
@@ -590,7 +607,7 @@ RETURNING true;
 -- name: FailFSMInstance :one
 UPDATE fsm_instances
 SET
-  current_state = '',
+  current_state = NULL,
```

Comment:
`''` is not able to be read back into a option.Option[schema.Ref]`, we need to set it to null